dev-docs/bidders/nanointeractive.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: bidder
+title: Nano Interactive
+description: Prebid Nano Interactive Bidder Adapter
+top_nav_section: dev_docs
+nav_section: reference
+biddercode: nanointeractive
+biddercode_longer_than_12: false
```

Comment:
Your bidder code is longer than 12 chars, which means that you need to set this to 'true' and should provide a block of hb_* variables. See IndexExchange as an example -- https://raw.githubusercontent.com/prebid/prebid.github.io/master/dev-docs/bidders/indexExchange.md